package.json

```diff
@@ -64,13 +64,16 @@
     "uuid": "2.0.3",
     "uuid-parse": "1.0.0",
     "validator": "10.8.0",
-    "webpack": "4.16.4"
+    "webpack": "4.16.4",
+    "webpack-cli": "^3.3.9"
```

Comment:
No up-carats please. Also, I'd really rather [run webpack from Node.js](https://masteringjs.io/tutorials/webpack/node) than pull in all of webpack-cli...

Reply:
Ah ok.  I'll remove it entirely and make a little script instead.

Also, I seem to remember some alternative "light" CLI.  I'll check just how light it really is.

Reply:
Never mind...  Looks like "webpack-command" has been deprecated.

Reply:
Ready.  Removed the dependency all together.